lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -477,3 +481,52 @@ MITIGATION_01_CRI_DCMAW:
     fail-with-no-ci:
       targetState: PYI_ANOTHER_WAY
 
+# Mitigation journey (02)
+MITIGATION_02:
+  response:
+    type: page
+    pageId: page-ipv-identity-document-start
+  events:
+    next:
+      targetState: MITIGATION_02_CRI_DCMAW
+    end:
+      targetState: MITIGATION_02_CRI_F2F
+
+MITIGATION_02_CRI_F2F:
```

Comment:
This was exploratory on the spike branch. Same session mitigation (for this ticket) is DCMAW only.